HISTORY.txt

```diff
@@ -1,5 +1,7 @@
 11.9.1:
  - Resolve editable packages on the local filesystem.
+ - Ensure lock hash does not change based on injected env vars.
```

Comment:
put this here because I'm assuming there will be a 11.9.1 release that will be pushed out, so best to put it here?